mappings/net/minecraft/client/font/BitmapFont.mapping

```diff
@@ -2,6 +2,29 @@ CLASS net/minecraft/unmapped/C_tfzxpuio net/minecraft/client/font/BitmapFont
 	FIELD f_mrmzkoex image Lnet/minecraft/unmapped/C_ayikuhxa;
 	FIELD f_pjauldcv LOGGER Lorg/slf4j/Logger;
 	FIELD f_syldtzsw glyphs Lnet/minecraft/unmapped/C_srmdcrvq;
+	CLASS C_qoeqnynz Definition1
```

Comment:
Is the `1` intentional? Looks like a typo

Reply:
oh yeah it is. [this enigma bug](https://github.com/QuiltMC/enigma/commit/92dab41ba0b23b8e23b9332d68734a281fdef7c4) meant I had to throw the `1` on there so it wasn't a conflict. just forgot to clean that one up :)